src/Plugin_Command.php

```diff
@@ -653,7 +655,7 @@ protected function get_item_list() {
 			}
 
 			$duplicate_names[ $name ][] = $file;
-			$items[ $file ]             = [
+			$items[ $file ] = [
```

Comment:
revert this change

Reply:
done 9693df372b720a7f6fb9686a92ba008ad9a80992